core/server/api_container/server/startosis_engine/package_io/package_io_test.go

```diff
@@ -1,8 +1,10 @@
 package package_io
 
 import (
+	"fmt"
```

Comment:
can we add an integration test?

Reply:
Sweet. Question, we had some problems where a dictionary inside the struct was still a dictionary; did we sort that out?

Reply:
Do we need an integration test? It's not a lot of effort to add one, but IMO the feature is well covered with the unit test, and integrations test are really slow so we should add one only when it's necessary

Reply:
And to your question, yes, a nested dictionary in the tope level JSON dict will become a nested struct inside the top level struct